README.md

```diff
@@ -46,14 +46,15 @@ In order to do so use the following command:
 git submodule add git@github.com:flowkey/UIKit-cross-platform.git UIKit && git submodule update --init --recursive UIKit
 ```
 
-### Android Studio Setup
+### Setup build environment
 
-1. Install [Android Studio](https://developer.android.com/studio/)
-2. Add SDKs in Android Studio
+1. Install [Cmake > 3.16](https://cmake.org/download/) and [Ninja](https://github.com/ninja-build/ninja/wiki/Pre-built-Ninja-packages)
```

Comment:
I'm surprised we need to install ninja separately because it's also installed by Android Studio. But this is fine I guess. Pretty sure you can install ninja using brew, which might be easier